src/electron/views/renderer-initializer.ts

```diff
@@ -85,6 +96,10 @@ const windowStateActions = new WindowStateActions();
 const scanActions = new ScanActions();
 const unifiedScanResultActions = new UnifiedScanResultActions();
 const cardSelectionActions = new CardSelectionActions();
+const detailsViewActions = new DetailsViewActions();
+const previewFeaturesActions = new PreviewFeaturesActions(); // not really used but needed by DetailsViewStore
```

Comment:
It feels like there are a lot of dependencies being pulled in just because of the use of the DetailsViewStore. I wonder if it would make more sense to do a PR prior to this one that separates out a `SettingsPanelStore` or something?

Reply:
(I almost wrote the same comment earlier in this review up at `automated-checks-view`)

Reply:
@dbjorge 
I kind of agree but I have made several PR's before this in preparation to bring the settings panel as cleanly as possible to AI-Android. I rather complete this PR and take on the refactor later. Thoughts?

Reply:
Separate PR seems reasonable to me!